module/PowerShellEditorServices/Commands/PowerShellEditorServices.Commands.psd1

```diff
@@ -77,7 +77,7 @@ FunctionsToExport = @('Register-EditorCommand',
                       'Out-CurrentFile',
                       'Join-ScriptExtent',
                       'Test-ScriptExtent',
-                      'psedit')
+                      'Open-EditorFile')
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
 CmdletsToExport = @()
```

Comment:
Should there be an entry to export `psedit` as an alias?  I guess for this module we don't need to worry about module auto-loading (since it is pre-loaded into PSIC).  Still, somehow feels better to declare everything in the manifest. :-)

Reply:
Oh interesting! didn't know you needed to list aliases in FunctionsToExport.

Thanks!

Reply:
Fixed!